### DISCUSSION TITLE
Can't load image for ImageBrush by relative path or using avares

### DISCUSSION THREAD

**Discussion by RagnarGothur** [AvaloniaUI/Avalonia#15491] [Q&A] [answered]
### Describe the bug

I want to use image as background for my UserControl.

```
<UserControl.Background>
	<ImageBrush Source="path/to/image.png" />
</UserControl.Background>
```

If I use absolute path like 

> C:\Users\Work\ProjectX\Assets\Images\X.png

everything goes well as the Image is correctly loaded as background.

But if I try relative path like 

> /MainMenu/X.png

or if I use avares like

> avares://ProjectX/Assets/Images/X.png

I constantly get **FileNotFoundException**

Interestingly, there is another resource request nearby that is working:
```
<UserControl.Styles>
	<StyleInclude Source="/MainMenu/MainMenuStyles.axaml"/>
</UserControl.Styles>
```

### To Reproduce

1) Create UserControl
2) Try set ImageBrush.Source using relative path or avares

### Expected behavior

Being able to load Bitmap from avares or relative path.

### Avalonia version

11.0.10

### OS

Windows

**Comment by maxkatz6** [comment, marked as answer]
Relative paths work, but asset needs to be registered as AvaloniaResource.
See https://docs.avaloniaui.net/docs/basics/user-interface/assets

**Reply by RagnarGothur** [reply]
I should have read the documentation more carefully... Thanks for your time!
